Editor/Bootstrap/RegionBootstrap.cs

```diff
@@ -11,7 +11,7 @@ namespace AmazonGameLift.Editor
     /// <summary>
     /// Keeps the region UI state and saves it to settings.
     /// </summary>
-    internal class RegionBootstrap
+    public class RegionBootstrap
```

Comment:
As with the previous PR, how much of this `internal` to `public` is necessary vs can be cleaned up with using the right namespaces

Reply:
Namespaces no, but I have just moved the file that needed it to be public and reverted it to internal.